tests/e2e/test_e2e.py

```diff
@@ -208,3 +211,19 @@ def test_prompt_manager_no_model_params_undeployed():
 
     with mgr.exec() as prompt:
         assert prompt.params is None
+
+
+def test_tracer_received_sigint_or_sigterm_cleanup():
+    test_trace_id = str(uuid.uuid4())
+    # Start the main.py script as a subprocess
```

Comment:
nit: this comment is referring to `main.py` but that is not the name of the script